gcslock.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+#
+# Copyright 2016 Google Inc.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#      http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file illustrates how to the use the gcslock global mutex
+# via the gsutil command in a shell script. It depends on installation
+# of the Google Cloud SDK (https://cloud.google.com/sdk/), which 
+# includes the gsutil command.
+
+if [ "$BUCKET" = "" ]
+then
+  echo "BUCKET env var undefined"
```

Comment:
what if I don't want to use an env var. could it more flexible if both `lock` and `unlock` funcs accepted the bucket as an arg?